src/cmd/package.go

```diff
@@ -203,6 +203,21 @@ var packageRemoveCmd = &cobra.Command{
 			message.Fatalf(err, lang.CmdPackageRemoveErr, err.Error())
 		}
 	},
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		var removeCandidates []string
+		// Get all the deployed packages
+		deployedZarfPackages, errs := cluster.NewClusterOrDie().GetDeployedZarfPackages()
+		if len(errs) > 0 && len(deployedZarfPackages) == 0 {
+			// Don't return an error with a completion, just return an empty list
+			return removeCandidates, cobra.ShellCompDirectiveDefault
```

Comment:
you can remove the errs, and the the whole statement with the if. If deployedZarfPackages is empty, the return at line 219 and 212 will be exactly the same :)

Reply:
Thanks! I modified as per your recommendation.